package/src/theme/defaultTheme.js

```diff
@@ -372,7 +402,7 @@ export const rawMuiTheme = {
         width: detailDrawerWidth
       },
       paperAnchorDockedRight: {
-        borderRight: "none"
+        border: "none"
```

Comment:
Think this needs to be applied to `paperAnchorLeft`, `paperAnchorDockedLeft`, `paperAnchorRight` as well

Reply:
resolved